### ISSUE TITLE
doesn't have the Noise92 dataset

### ISSUE THREAD

**Issue by way2coder** [Audio-WestlakeU/FN-SSL#1]
I donn't know where to download the Noise92 dataset,could please give a link

**Comment by wangyabo123** [comment]
This is the official link: http://spib.linse.ufsc.br/noise.html. And you can download the noise data we used from the https://pan.baidu.com/s/1gLxrYT68UQ_5ff0OcHv-1A?pwd=osu9

**Comment by way2coder** [comment]
Thank u very much. Have a good day

**Comment by way2coder** [comment]
This download link is out of date, could you please send this link again, thank you very much!

**Comment by wangyabo123** [comment]
ok, this is the new link：https://pan.baidu.com/s/1SsuGkf8ZIzKYCkrHurN--g?pwd=yoj2

**Comment by way2coder** [comment]
Thank you^_^

**Comment by way2coder** [comment]
Hi, I have a question, in this paper, the number of utterances for 3 parts is 166,816, 992, and 5,000, respectively. Does it mean the configuration of the data_num in Simu.py to 166,816?  992, and 5,000? Is this a one-to-one correspondence? Thank you^-^

**Comment by wangyabo123** [comment]
Sorry for the late respond, you can set the configuration as the paper shown, also you can change the number of dataset in your experiment because the number of the dataset not have special meanings.

**Comment by SekiroRong** [comment]
This download link is out of date, could you please send this link again, thank you very much!

**Comment by wangyabo123** [comment]
Hi, you can download the noise data through the  official link: http://spib.linse.ufsc.br/noise.html.

**Comment by SekiroRong** [comment]
Thank you for replying. However, the noise data in the official link is in format of **.mat**, should I convert those files into **.wav** format?

**Comment by wangyabo123** [comment]
yes, if you want to use the fn-ssl code, you need to convert those files into wav format.
